modules/retail/pages/retail-uyuni-server-setup.adoc

```diff
@@ -0,0 +1,291 @@
+[[retail-server-setup]]
+= Retail Uyuni Server Setup
+
+This section covers {productname} {smr} Server setup, using these procedures:
+
+* Start {productname} setup with {yast}
+* Create the main administration account with the {productname} {webui}
+* Name your base organization and add login credentials
+* Synchronize the openSUSE Leap channel
```

Comment:
If you're using a process/procedure IA (which, iirc, we do a lot in the Retail Guide), then you should make sure that the steps in the process exactly match the the titles of the procedures. Otherwise, it can be super confusing.